### DISCUSSION TITLE
No suggestion appeared when using GitHub Copilot

### DISCUSSION THREAD

**Discussion by haitranwang** [community/community#20447] [Copilot Conversations]
Hello
I joined the free trail plan, Installed, enabled GitHub Copilot and followed the step in "Seeing your first suggestion" in the document but there is no suggestion appear...

Im using VS Code Version: 1.69.1 on MacOS.

<img width="814" alt="image" src="https://user-images.githubusercontent.com/104315484/178785737-79f26345-e154-40b0-875c-c1703113a43c.png">

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/104315484/178785963-f0cc2388-cf83-42a3-9935-7bbfa7af2201.png">

Can anyone help me?

**Comment by FahdSeddik** [comment]
Hey! @haitranwang ,
This seems to be a similar issue to the one stated [here](https://github.com/github-community/community/discussions/14117) . Hope it helps!

**Reply by FahdSeddik** [reply]
If it helped, make sure to mark it as an answer so people would see it easier.

**Reply by haitranwang** [reply]
Hi @FahdSeddik 

I already read it 
Here is my bottom tool bar in VSC, seem ok and I also re-install several times the extension but Copilot still not working :( 

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/104315484/178787458-74887dc5-65e9-4ec2-afd8-696aae6e12e7.png">

**Reply by FahdSeddik** [reply]
Maybe you can check whether you have received the copilot's invitation first. If so, just enter vscode and reload the copilot.

**Reply by FahdSeddik** [reply]
Also, do you happen to use proxy?

**Reply by FahdSeddik** [reply]
Here is a video on macos. make sure you follow the steps [here](https://youtu.be/JlDan-XCG7U).

**Reply by haitranwang** [reply]
I reloaded the extension several times also.
Also, do you happen to use proxy? => No, I used to use Copilot before and everything was fine but since Copilot switched to a paid subscription I had to re-register and now it doesn't work anymore

**Reply by FahdSeddik** [reply]
I suggest you contact GitHub support [here](https://support.github.com/contact/account).

**Reply by haitranwang** [reply]
Hi @FahdSeddik 
Yes thank you, I created a ticket: https://support.github.com/ticket/personal/0/1705348
